src/script/auth/page/SingleSignOn.tsx

```diff
@@ -156,20 +156,22 @@ const SingleSignOnComponent = ({hasDefaultSSOCode}: Props & ConnectedProps & Dis
 
       amplify.subscribe(WebAppEvents.LIFECYCLE.SSO_WINDOW_CLOSED, () => {
         onChildWindowClose();
-        reject(new BackendError({code: 500, label: BackendError.LABEL.SSO_USER_CANCELLED_ERROR}));
+        reject(new BackendError('', SyntheticErrorLabel.SSO_USER_CANCELLED_ERROR, StatusCodes.INTERNAL_SERVER_ERROR));
```

Comment:
Probably something obvious, but what's the empty string here and in the following ones?

Reply:
error message